docs/api/graph/tick.md

```diff
@@ -54,3 +54,24 @@ way of doing that is to disable the Y-axis labels by setting the tick label mode
 @@@ atlas-graph { show-expr=false }
 /api/v1/graph?e=2012-01-01T00:00&q=name,sps,:eq,(,nf.cluster,),:by&s=e-1w&tick_labels=off
 @@@
+
+## Offset Labels
+
+In situations where a measurement with a large deviation from zero has small deviations
+during the query range, ticks may be labeled with offsets in order to fit the labels in
+the layout. This only happens when the [axis bounds] are configured for automatic layout
```

Comment:
It happens when the number of significant digits required to render the delta is too large, automatic or explicit bounds doesn't matter. It is just uncommon anyone explicitly sets bounds in a way that would trigger it.